index.html

```diff
@@ -0,0 +1,185 @@
+      </ul>
+    </div>
+    </section>
+    <!--section 1 of the portfolio card starts-->
+    <section class="sectiontemplate">
+      <img class="img1" src="img/Portfolio1.svg" alt="gym">
+      <div class="Section2Text">
+      <h2 class="face">Facebook 360</h2>
+        <ul class="list1">
+          <li class="can">Canopy</li>
+          <li class="special">Back End Dev</li>
+          <li class="special">2015</li>
+        </ul>
+      <p>A daily selection of privately personalized reads; 
+      no accounts or sign-ups required.</p>
+        <ul class="list2">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul> 
+    <button class="btn">See project</button>
+    </div>
+    </section>
+    <!--section 2 of the portfolio cards starts-->
+    <section class="sectiontemplate">
+      <img class="img1" src="img/Portfolio2.svg" alt="gym">
+      <div class="Section3Text">
+        <h2>Uber Navigation</h2> 
+        <ul class="list1">
+          <li class="can">Canopy</li>
+          <li class="special">Back End Dev</li>
+          <li class="special">2015</li>
+        </ul>
+      <p>A daily selection of privately personalized reads; 
+        no accounts or sign-ups required.</p>
+        <ul class="list3">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+      <button class="btn">See project</button>
+      </div>
+    </section>
+    <!--section 3 of the portfolio card starts-->
+    <section class="sectiontemplate">
+      <img class="img1" src="img/Portfolio3.svg" alt="gym">
+      <div class="SectionText">
+        <h2>Tonic</h2>
+        <ul class="list1">
+          <li class="can">Canopy</li>
+          <li class="special">Back End Dev</li>
+          <li class="special">2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no 
+          accounts or sign-ups required.</p>
+        <ul class="list2">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul> 
+      <button class="btn">See project</button>    
+      </div>
+    </section>
+    <!--section 4 of the portfolio card starts-->
+    <section class="section1template">
+      <img class="img2" src="img/Portfolio4.svg" alt="gym">
+        <div class="Section1Text">
+          <h2 class="multi">Multi-Post Stories</h2>  
+        <ul class="list1">
+          <li class="can">Canopy</li>
+          <li class="special">Back End Dev</li>
+          <li class="special">2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required.</p>
+        <ul class="list3">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See project</button>
```

Comment:
- [ ] As required, please add button actions for all you `see project and get my resume` button as shown in this image below ⏬ 
<img width="683" alt="action" src="https://user-images.githubusercontent.com/79058364/159116860-d1f3610d-f31a-49a4-a88a-5ec2a165e340.png">